jina/drivers/rank/aggregate/__init__.py

```diff
@@ -105,9 +104,9 @@ def _apply_all(self, docs: 'DocumentSet',
         :return:
         """
 
-        match_idx = []  # type: List[Tuple[UniqueId, UniqueId, UniqueId, float]]
-        query_meta = {}  # type: Dict[UniqueId, Dict]
-        match_meta = {}  # type: Dict[UniqueId, Dict]
+        match_idx = []  # type: List[Tuple[str, str, str, float]]
```

Comment:
Why are these types assigned behind a comment?

Reply:
https://realpython.com/lessons/type-comments/

Reply:
Yes, but why are they comments in this case? They only reason to do that is if we want to support Python 2. Anyway, not a big deal